### PR TITLE
1.0.2 release (develop -> master)

### PR DESCRIPTION
**Pull request by Xiangs18**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
 
+### Version 1.0.3
+* The MongoDB clients have been updated to the most recent version.
+* Added the mongo-retrywrites configuration setting in deployment.cfg.templ, defaulting to false.
+* Migrated from Travis CI to GitHub Actions workflows.
+* Updated Python version to 3.9.19.
+
+### Version 1.0.2
+- Add a cache for bad tokens so they aren't looked up over and over. Maxes out at 10000, then throws out the oldest bad token.
+
```
Comment (MrCreosote) [inline RELEASE_NOTES.md:11]: @briehl it looks like 1.0.2 was never released, can you confirm? If so we should merge the 1.0.3 changes into 1.0.2

**Review by MrCreosote — COMMENTED** [review]
changes look reasonable based on spot checks other than a possible version skip

File: RELEASE_NOTES.md (hunk 1 above)
Comment (Xiangs18) [reply RELEASE_NOTES.md:11]: I rolled back and checked:

{"gitcommithash":"a2ed4cb88120aeb10a295919cb0fba85e13d462d","servertime":1732243387039,"service":"Notification Feeds Service","version":"1.0.2"}

Comment (briehl) [reply RELEASE_NOTES.md:11]: I can confirm 1.0.2 was never released in production. Not sure why.

Comment (Xiangs18) [reply RELEASE_NOTES.md]: https://github.com/kbase/feeds/pull/90

**Review by briehl — APPROVED** [review]

**Pull request merged**